php/Plugin.php

```diff
@@ -88,11 +89,77 @@ public function require_deprecated() {
 		}
 	}
 
+	/**
+	 * Requires the API functions.
+	 */
+	private function require_api() {
+		require_once __DIR__ . '/BlockApi.php';
+	}
+
 	/**
 	 * Requires helper functions.
 	 */
 	private function require_helpers() {
-		require_once __DIR__ . '/Helpers.php';
-		require_once __DIR__ . '/BlockApi.php';
+		if ( $this->is_plugin_conflict() ) {
+			add_action( 'admin_notices', [ $this, 'plugin_conflict_notice' ] );
+		} else {
+			require_once __DIR__ . '/Helpers.php';
+		}
+	}
+
+	/**
+	 * Gets whether there is a conflict from another plugin having the same functions.
+	 *
+	 * @return bool Whether there is a conflict.
+	 */
+	public function is_plugin_conflict() {
+		return function_exists( 'block_field' ) || function_exists( 'block_value' );
+	}
+
+	/**
+	 * An admin notice for another plugin being active.
+	 */
+	public function plugin_conflict_notice() {
+		$plugin_file      = 'block-lab/block-lab.php';
```

Comment:
It's a real shame that we've got to mention Block Lab by name in GCB. This isn't scalable to every plugin that ever uses `block_field()` or `block_value()`.

Reply:
Yeah, it's dirty 😄 

This is one of the problems in changing the name of a plugin and not forcing users to manually rename `block_field()` in every template.